tools/generator/src/PBXTarget+Extensions.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import XcodeProj
+
+public extension PBXTarget {
+    var buildableName: String {
+        return productName ?? name
+    }
+
+    func createBuildableReference(
+        referencedContainer: XcodeProjContainerReference
+    ) -> XCScheme.BuildableReference {
+        return .init(
+            referencedContainer: "\(referencedContainer)",
+            blueprint: self,
+            buildableName: buildableName,
+            blueprintName: name
+        )
+    }
+
+    var schemeName: String {
+        // GH371: Update XcodeProj to support slashes in the scheme name.
+        // The XcodeProj write logic does not like slashes (/) in the scheme
+        // name. It fails to write with a missing folder error.
+        return buildableName.replacingOccurrences(of: "/", with: "_")
+    }
+
+    var isTestable: Bool {
+        return productType?.isTestBundle ?? false
+    }
+
+    var isLaunchable: Bool {
+        return productType?.isExecutable ?? false
```

Comment:
We instead need a `isLaunchable` on `PBXProductType` that we reference here, as `isExecutable` includes a lot of non-launchable things, mainly frameworks. `isExecutable` just means it has an executable Mach-O binary (versus static or none).

Reply:
I updated my code to use the `isLaunchable` on `PBXProductType`.